### ISSUE TITLE
Clean Loadout ALL pylons removed Issue-Tumbling jet /Dead Engine

### ISSUE THREAD

**Issue by Shamu06** [heclak/community-a4e-c#417]
**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior:  This is in the latest version down loaded today 18JUL21. It does NOT exist in last 1.4.2. I did not test any versions other than these two. 
2. Start DCS, 
3. Create mission with A-4E-C to **start on runway**
4. change loadout by REMOVING ALL pylons
5. Save
6. Start mission

**Expected behavior**
One of two things will happen OR BOTH. I have had only a. but now I am getting b. only

a. Jet is on runway but power dies and jet shuts down. Cant be restarted
b. Jet starts tumbling wildly .....or at a minimum will just tip over like a dead cow as engine dies

**Screenshots**
If it does it for you you will KNOW it happens and not need a screen shot. Trust me.

**Software Information (please complete the following information):**
 - DCS Version: 2.7 latest
 - A-4E Version: Community A-4E-C v2.0.0-beta-5.1

As I stated above - Community A-4E-C v1.4.2 acts/performs normally and does not do this. I tested NO other versions than these two. Since  v2.0.0-beta-5.1 is newer it well may perform normally in all other respects than reported/known bugs and 1.4.2 since it is not newer may not perform in all respects in DCS 2.7.  I have no clue - I only tested normal take offs after configuring all axis for flight and assigning gear, flaps and speedbrake functions - i.e just pure flight - a simple hop around the airfield. NO other functions tested.

**Additional context**
ALL IS ABOVE

SHAMU06

**Comment by 08jne01** [comment]
Do you have the CH-53 installed?

**Comment by Shamu06** [comment]
IRONICALLY I do

**Comment by Shamu06** [comment]
If that is causing the issue I will UNinstall it. It is a pain in the ass anyway

**Comment by Shamu06** [comment]
Flying helicopters is like masturbating anyway. It is fun but you dont want anyone to know you do it.

**Comment by 08jne01** [comment]
> If that is causing the issue I will UNinstall it. It is a pain in the ass anyway

Yes sir, their mod configuration triggers a DCS bug which corrupts the memory in DCS. This is noticeable with the issues you mention.

**Comment by Shamu06** [comment]
The Stallion is NUKED. Ops now normal.  CHEERS and THANKS 08jne01 !!!!

**Comment by callmepartario** [comment]
just want to check in and see if the EFM will now give a shutdown prompt when the CH-53E is installed, i know we talked about that

**Comment by 08jne01** [comment]
Added a protection to this so that it warns the user that these are incompatible. We do need to check that the strange behaviour is occurring before releasing in the off chance ED actually fixed this.

**Comment by searching46dof** [comment]
To fix incompatibility with CH47 Chinook mod, modify Saved Games\DCS\Mods\aircraft\CH7 Chinook\FM\Ch47_Chinook_config.lua  as:

![Image](https://github.com/user-attachments/assets/243bbd90-10ca-46c3-b0b8-fbab9dd5f799)

To fix incompatibility with CH53 mod, modify Saved Games\DCS\Mods\aircraft\Sokorsky CH-53E Super Stallion\FM\CH_53_config.lua as:

![Image](https://github.com/user-attachments/assets/d7b8ec8d-5f98-4a26-8b8c-0ebb6dd9025e)

**Comment by searching46dof** [comment]
To fix incompatibility with AH-6J, there are files missing in AH-6J v3.5 and v4.0
Extract v3.5 or v4.0 over v2.0 (https://www.digitalcombatsimulator.com/en/files/3314815/)

![Image](https://github.com/user-attachments/assets/662f0f74-4890-429f-abab-8fb027205a5d)

**Comment by searching46dof** [comment]
Can you add a configuration setting to disable the check for the "AH-6J Little Bird", "CH-47 Chinook", and "CH-53E Super Stallion" mods? Perhaps something as simple as a variable in a lua.
I have patched them such that there is no incompatibility and they can coexist.
When playing in 2D you can easily select "ignore" in the warning dialog to continue.
However when playing in VR the warning dialog is only displayed on the 2D screen and not in the VR headset's view.
